docs/io/output/rpacket_tracking.ipynb

```diff
@@ -43,12 +43,7 @@
    "id": "4b0de6ca",
```

Comment:
<p>Remove trailing "###" - in markdown you need <code>#</code>s only at beginning  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2073/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>